at_utils/pubspec.yaml

```diff
@@ -10,8 +10,14 @@ environment:
 dependencies:
   logging: ^0.11.4
   yaml: ^2.2.0
-  crypto: ^2.1.5
-  at_commons: ^1.0.1+5
+  crypto: ^3.0.0
+  at_commons:
+    git:
+      url: https://github.com/atsign-foundation/at_tools.git
+      path: at_commons
+      ref: dev_env
```

Comment:
please modify ref after merge

Reply:
In prod_staging and prod, we'll point to pub.dev dependencies.